data_steward/tools/purge_hpo_data.py

```diff
@@ -0,0 +1,100 @@
+"""
+Empty all CDM tables associated with one or more HPO sites from an EHR dataset
+
+This can be used to exclude sites' data from the EHR dataset before 
+generating the unioned dataset which will be used for the CDR.
```

Comment:
I think I'm right.  You're using this doc string as the command line description for the module, correct?  I like the idea.  It might be a good idea to explicitly state that this module does not delete the empty tables.  I know the first line states you are emptying the CDM tables, but since I was just skimming the comment I didn't pay **close** attention.  That's why it **might** help to explicitly state the tables are not deleted.  Because that was my first question.  Why not just delete the tables?  It's probably something in the generation process that I'm just not familiar with.

Reply:
Thanks- the docstring is now more explicit.